templates/docs/patterns/hero/index.md

```diff
@@ -17,12 +17,105 @@ The hero pattern is composed of the following elements:
 
 | Element              | Description                                                              |
 | -------------------- | ------------------------------------------------------------------------ |
-| Title (**required**) | <code>h1</code> title text                                               |
-| Subtitle             | <code>h2</code> subtitle text                                            |
-| Description          | <code>p</code> description text                                          |
+| Title (**required**) | `h1` title text                                                          |
+| Subtitle             | `h2` subtitle text                                                       |
```

Comment:
Technically it's not an `h2`. It's a subtitle that uses h2 style (but it's not a heading). I know it's a nuance that may be not necessarily needed to be understood by users (calling it `h2` may be an OK simplification), but on the same time it is a bit misleading.

Maybe something like:

```
title text (to be placed in `h1` heading)
subtitle text (using `h2` style)
description text (one or more paragraphs)
```

Reply:
Revised